bindings/nodejs/lib/types/block/output/output.ts

```diff
@@ -35,15 +35,15 @@ enum OutputType {
  * The base class for outputs.
  */
 abstract class Output /*implements ICommonOutput*/ {
-    readonly amount: string;
+    readonly amount: NumericString;
```

Comment:
Don't we use `u64` for things like this ?

Reply:
no its not bigint yet, thats 2.0